kebab_city_tour/src/router.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
```

Comment:
```suggestion
import { createBrowserRouter } from 'react-router-dom';
```


This is new better way

Reply:
Well, I changed that and it was mad at me 
![Zrzut ekranu 2024-10-11 221848](https://github.com/user-attachments/assets/b7ba735b-3a8f-4b0b-8ab8-a53a70b2e10a)

Reply:
❌✔️